nixos/tests/tus-curl-upload.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Would be better to have this script inside the test file

Reply:
I disagree: Then I can't easily use it for local/interactive testing when it doesn't work.

Reply:
having the file tus-curl-upload.sh just in nixos/test/ seems messy, maybe create a sub directory nixos/tests/tusd and put everything in there?

Reply:
Ah, I see, yes. Done, I did that move.